test/cli/gold_standard/char_both/25e-3.gff.stdout

```diff
@@ -1,8 +1,8 @@
 I/O options:
   database file   : 512_simSeq1_25e-3.fa
   query file      : 512_simSeq2_25e-3.fa
-  alphabet        : protein
-  output file     : protein_both/25e-3.gff
+  alphabet        : char
```

Comment:
First commiting the copy is a nice trick to see the difference, will remember it :)

Reply:
> Is it python based testing because the app is using seqan2?

Yes! I refactored the old tests to be able to run them separately. 

I wanted to also increase the code coverage, as I noticed that the reverse-complement strand "mode" wasn't executed by the tests.

Furthermore, I have some more refactoring of the app that I wanted to validate that it did not change the app logic.